Makefile

```diff
@@ -58,3 +58,27 @@ integration:
 		--log-cli-level=INFO \
 		$(TESTS)/integration \
 		$(ARGS)
+
+# Run scenario tests
+scenario:
```

Comment:
I think that we should start collapsing unit and scenario tests into unit.

Reply:
combined them both under `unit` in the Makefile